spec/concepts/reservations/use_cases/update_reservation_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Reservations::UseCases::UpdateReservation do
+  describe '.call' do
+    context 'when passed status is confirmed' do
+      subject(:update_reservation) { described_class.new(reservation, 'confirmed').call }
+
+      let(:reservation) { create(:reservation) }
+      let(:params) { { status: 'confirmed' } }
```

Comment:
you don't use this variable in this context

Reply:
yes, I had some problems with this test, and during debugging I changed this to string and forgot about it:)